CHANGE_LOG

```diff
@@ -85,6 +89,9 @@ Misc Enhancements:
 * PR #2419: Add support for returning enums in vectorize 
 * PR #2421: Add copyright notice for Intel contributed files.
 * PR #2422: Patch code base to work with np 1.13 release
+* PR #2448: Adds in warning message when using 'parallel' if cache=True
+* PR #2450: Add test for keyword arg on .sum-like and .cumsum-like array
+            methods
```

Comment:
The "be considered experimental." line has the same problem.  It shouldn't be indented